programs/ssl/ssl_pthread_server.c

```diff
@@ -331,6 +334,14 @@ int main( void )
     mbedtls_ssl_cache_context cache;
 #endif
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", ret );
+        goto exit;
+    }
+#endif
```

Comment:
for `ssl_pthread_server`, I would add the platform context to `thread_info_t` , and protect it with a mutex